### PR TITLE
Add migration guide 

### PR DESCRIPTION
**Pull request by jonasNobile**

### CODE REVIEW

File: docs/admin/releases/v0.8.0-alpha.7.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* editor: sortable blocks by @matej21 in https://github.com/contember/admin/pull/76
+* select field fixes by @matej21 in https://github.com/contember/admin/pull/82
+* Fix multiselect by @matej21 in https://github.com/contember/admin/pull/83
+* Admin server: project groups support by @matej21 in https://github.com/contember/admin/pull/78
+* Datagrid update by @matej21 in https://github.com/contember/admin/pull/81
+* Collaboration server by @VojtaStanek in https://github.com/contember/admin/pull/85
+* Collaboration server: fix crash by @VojtaStanek in https://github.com/contember/admin/pull/88
+* Datagrid: add select cells by @matej21 in https://github.com/contember/admin/pull/87
+* password reset and OTP support by @matej21 in https://github.com/contember/admin/pull/60
+* Update design using CSS custom properties by @attitude in https://github.com/contember/admin/pull/84
```
Comment (VojtaStanek) [inline docs/admin/releases/v0.8.0-alpha.7.mdx:24]:
```suggestion

### New Features
* New design [#84](https://github.com/contember/admin/pull/84)
* Password reset support [#60](https://github.com/contember/admin/pull/60)
* One-time password support [#60](https://github.com/contember/admin/pull/60)
* Collaboration server in admin-server [#85](https://github.com/contember/admin/pull/85)
* Editor: Sortable blocks and major refactoring including new API for adding custom blocks and bug fixes [#71](https://github.com/contember/admin/pull/71), [#76](https://github.com/contember/admin/pull/76), [#82](https://github.com/contember/admin/pull/82)
* Datagrid: `HasOneSelectCell` and `HasManySelectCell` [#87](https://github.com/contember/admin/pull/87)
* Datagrid: Remembers selected filters and page when navigating back [#81](https://github.com/contember/admin/pull/81)


### Fixes
* Various select field fixes [#82](https://github.com/contember/admin/pull/82), [#83](https://github.com/contember/admin/pull/83)
* `LogoutLink` in Safari [#71](https://github.com/contember/admin/pull/71)

```

**Review by VojtaStanek — CHANGES_REQUESTED** [review]
The migration guide looks awesome!

File: docs/admin/releases/v0.8.0-alpha.7.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+- We no longer support `<AltPersistButton />`. Remove it or change it to `<PersistButton />`.
+
+#### Tenant pages
+
+- Change the `<GenericPage />` that wraps the tenant pages to `<Page />`.
+
+### Recommended changes
+
+Remove unnecessary component `<Box />`.  We added a new component `<Section />` which should be used instead of `<Box />` to define groups of fields.
```
Comment (attitude) [inline docs/admin/releases/v0.8.0-alpha.7.mdx:128]: There few things to be noted, here's my suggestion (maybe in bullets for better readability):

 "... split groups of fields on the page into logical page sections. Sections with titles are also listed within the page tabs navigation. Sections are not meant to be nested within other sections and should only be used to define top level sections of the page. Sections are required to be parent elements to dimensions not the other way around due to styling limitations."

**Review by attitude — CHANGES_REQUESTED** [review]
Sections are new and there are some other changes required when migrating. See the comment. Besides that, I approve merging.

**Review by honzasladek — APPROVED** [review]
Looks great! 🎉

**Review by VojtaStanek — APPROVED** [review]

**Pull request merged**